pkg/agent/openflow/client.go

```diff
@@ -30,7 +30,12 @@ import (
 	"github.com/vmware-tanzu/antrea/third_party/proxy"
 )
 
-const maxRetryForOFSwitch = 5
+const (
+	maxRetryForOFSwitch = 5
+	// Due to the message size of openflow 1.3 and implementation of Service in Antrea, the maximum Endpoint that Antrea
+	// can support now is 800. If the Endpoints of Service exceed 800, the exceeding Endpoints will be ignored.
+	maxEndpoints = 800
```

Comment:
so we went from 500 to 800? This is a bit underwhelming to be honest, albeit still an improvement

Reply:
Yeah, still an improvement.